doc/issue_impact.md

```diff
@@ -0,0 +1,25 @@
+# Issue Impacts
+
+axe-core assigns an impact according to our assessment of the likely impact of an issue on a user with a disability that would be affected by this issue.
+
+In any given context the actual impact for the user could be lower and in some instances, it could be higher.
+
+For this reason, we encourage users of tools to evaluate each individual issue and assess the impact in the context of their application or content.
+
+## Definitions
+
+### Minor
+
+Considered to be a nuisance or an annoyance bug. Prioritize fixing if the fix only takes a few minutes and the developer is working on the same screen/feature at the same time, otherwise the issue should not be prioritized. Will still get in the way of compliance if not fixed. Should be very infrequent.
+
+### Moderate
+
+Results in some difficulty for people with disabilities, but will generally not prevent them from accessing fundamental features or content. Users may be frustrated and abandon non-critical workflows. Prioritize fixing in this release, if there are no higher-priority issues. Will get in the way of compliance if not fixed. Should be fairly common.
+
+### Serious
+
+Results in serious barriers for people with disabilities, and will partially or fully prevent them from accessing fundamental features or content. People relying on assistive technologies will experience significant frustration and may abandon essential workflows. Issues falling under this category are major problems, and remediation should be a priority. Should be very common.
+
+### Critical
+
+Results in blocked content for people with disabilities, and will definitely prevent them from accessing fundamental features or content. This type of issue puts your organization at risk. Prioritize fixing as soon as possible, within the week if possible. Remediation should be a top priority. Should be infrequent.
```

Comment:
> This type of issue puts your organization at risk

At risk of what?

Reply:
I want to be vague because the risk depends very much on the geographic location and/or the industry.